src/common/components/Tooltip/Tooltip.tsx

```diff
@@ -1,24 +1,103 @@
-import * as React from 'react'
+import React, { FunctionComponent } from 'react'
+import ArrowToolTip from '../../assets/icons/ico-triangle.svg'
+import cn from 'classnames'
+import CSS from 'csstype'
 
-import Typography from '../../../common/components/Typography/Typography'
-import { colors } from '../../theme/colors'
-
-import * as Styles from './Tooltip.styles'
-
-interface TooltipProps {
+export type TooltipProps = {
+  type: 'left' | 'right' | 'top' | 'bottom'
+  classnames?: string
   content: string
+  width: number
 }
 
-const Tooltip: React.FC<TooltipProps> = ({ children, content }) => {
+const Tooltip: FunctionComponent<TooltipProps> = ({
+  type,
+  classnames,
+  content,
+  width,
+  children,
+}) => {
+  const styles = {
+    top: {
+      width: `${width}px`,
+      bottom: '150%',
+      left: `calc(50% - ${width / 2}px)`,
+    },
+    bottom: {
+      width: `${width}px`,
+      top: '150%',
+      left: `calc(50% - ${width / 2}px)`,
+    },
+    right: {
+      width: `${width}px`,
+      left: 'calc(100% + 10px)',
+    },
+    left: {
+      width: `${width}px`,
+      right: 'calc(100% + 10px)',
+    },
+  }
+  const arrow_styles = {
```

Comment:
Can we use `tailwind` instead of adding the CSS here?

Reply:
I'm not sure how to use tailwindcss for that. with calc function